### ISSUE TITLE
cleanup

### ISSUE THREAD

**Issue by changken1** [QTIM-Lab/qtim_ChallengePipeline#1]
https://github.com/QTIM-Lab/qtim_ChallengePipeline/blob/4d79750fb1d75d8304efe413011c417d80c3d613/model.py#L334

convolutions, dropout, and filter_shape arguments are not used

**Comment by changken1** [comment]
https://github.com/QTIM-Lab/qtim_ChallengePipeline/blob/4d79750fb1d75d8304efe413011c417d80c3d613/model.py#L342

Add batchnorm to first conv layer (currently there are only batch norm after the second conv layer)
